src/components/ui/animations/StarAnimation.tsx

```
@@ -5,37 +5,35 @@ const { Star } = MiscellaneousIcons;
+                y: (i % 2 === 0 ? -1 : 1) * 40,
+              }}
+              transition={{ duration: 0.7, delay: i * 0.1 }}
+              className="absolute z-10"
+            >
+              <Star
+                className="h-6 w-6 text-yellow-400 drop-shadow-lg"
+                fill="currentColor"
+              />
+            </motion.div>
+          ))}
+        </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)